jenkins-backup.sh

```diff
@@ -58,12 +58,15 @@ function main() {
   fi
 
   rm -rf "${ARC_DIR}" "{$TMP_TAR_NAME}"
-  for plugin in plugins jobs users secrets nodes; do
+  for plugin in plugins jobs users secrets nodes labels; do
     mkdir -p "${ARC_DIR}/${plugin}"
   done
 
   cp "${JENKINS_HOME}/"*.xml "${ARC_DIR}"
 
+  cp "${JENKINS_HOME}/.owner" "${ARC_DIR}" 
+  ls "${JENKINS_HOME}/plugins" > ${ARC_DIR}/content_of_plugins_folder.txt
```

Comment:
@cafhach **[QUESTION]** Is this really necessary?

The subject of this PR is the addition of the `labels` directory, I don't think this is relevant